runtime/sema/check_while.go

```diff
@@ -118,7 +118,7 @@ func (checker *Checker) VisitBreakStatement(statement *ast.BreakStatement) ast.R
 
 	// check statement is inside loop
 
-	if !checker.inLoop() {
+	if !(checker.inLoop() || checker.inSwitch()) {
```

Comment:
```suggestion
	// break is only allowed to be inside of a for-loop or a switch-case,
	// otherwise return error
	if !(checker.inLoop() || checker.inSwitch()) {
```

Reply:
Fixed in a2b0c9e34a5a4dc8fe37001eb828fb851d870312